Sources/docc/DocCDocumentation.docc/adding-images.md

```diff
@@ -0,0 +1,40 @@
+# Adding Images to Your Content
+
+Elevate your content's visual appeal by adding images.
+
+## Overview
+
+DocC extends Markdown's image support so you can provide appearance and 
+display scale-aware versions of an image. You use specific components to create image filenames, and DocC uses the most appropriate version of the image when displaying your documentation.
+
+![An image of a filename that's split into four labeled sections to highlight the individual components. From left to right, the components are the image name, the appearance mode, the display scale, and the file extension.](docc-image-filename)
+
+| Component | Description |
+| --- | --- |
+| Image name | **Required**. Identifies the image within the documentation catalog. The name must be unique across all images in the catalog, even if you store them in separate folders. |
+| Appearance | **Optional**. Identifies the appearance mode in which DocC uses the image. Add `~dark` directly after the image name to identify the image as a dark mode variant. |
+| Display scale | **Optional**. Identifies the display scale at which DocC uses the image. Possible values are `@1x`, `@2x`, and `@3x`. When specifying a display scale, add it directly before the file extension. |
```

Comment:
The sentence, "Possible values are `@1x`, `@2x`, and `@3x`." is inaccurate. We don't require people to add the `@1x` attribute to filenames for standard, non-Retina, images. Would also be good here to say something like, "To best support your readers, provide images in standard resolution and `@2x` scale." or something to that effect.

Reply:
Resolved https://github.com/apple/swift-docc/pull/742/commits/ec67b2c642577028a3973f18087d7b22a936a8ff